src/viking.js

```diff
@@ -1,19 +1,93 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor (health,strength){
+    this.health=health
+    this.strength=strength
+  }
+  attack(){
+    return this.strength
+  }
+  receiveDamage(damage){
+    this.health -= damage
+  }
+}
+
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor (name, health, strength) {
+    super (health, strength)
+    this.name = name
+    this.health = health
+    this.strength = strength
+  
+  }
+  receiveDamage(damage) {
+  this.health -= damage
+  if (this.health <= 0 ) {
+    return `${this.name} has died in act of combat`
+  }
+  return `${this.name} has received ${damage} points of damage`
+  }
+  battleCry(){
+    return "Odin Owns You All!"
+  }
+}
 
 // Saxon
-class Saxon {}
+class Saxon extends Soldier{
+  //attack(){
```

Comment:
Para crear e inicializar un objeto hay que hacer uso del constructor y no debe de haber mas de uno en cada clase 😉 
```suggestion
 constructor(healthArg, strengthArg) {
    super(healthArg, strengthArg);
  }
```